JamesIves/github-pages-deploy-action
# Adding tag option to action

## Description

I found out that in some cases, pushing to the doc repo using a tag is convenient.

## Testing Instructions

In theory, it should work by providing the key option `tag`.

```
- name: Deploy 🚀
  uses: JamesIves/github-pages-deploy-action@v4.3.3
  with:
    branch: gh-pages # The branch the action should deploy to.
    folder: build # The folder the action should deploy.
    tag: 'v0.1'
```

## Additional Notes

* This is my first time using TypeScript, so watch out for the errors.

* This PR closes #1147
